extensions/resource-deployment/src/interfaces.ts

```diff
@@ -261,6 +261,11 @@ export interface DynamicEnablementInfo {
 	value: string
 }
 
+export interface ValueProviderInfo {
+	providerId: string,
```

Comment:
here too, without this your recent change won't work.

Reply:
(responding in other comment)